<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>675</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>675</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>675</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QSlider:disabled {
  background: #19232D;
}

QSlider:focus {
  border: none;
}

QSlider::groove:horizontal {
  background: #32414B;
  border: 1px solid #32414B;
  height: 4px;
  margin: 0px;
  border-radius: 4px;
}

QSlider::groove:vertical {
  background: #32414B;
  border: 1px solid #32414B;
  width: 4px;
  margin: 0px;
  border-radius: 4px;
}

QSlider::add-page:vertical {
  background: #1464A0;
  border: 1px solid #32414B;
  width: 4px;
  margin: 0px;
  border-radius: 4px;
}

QSlider::add-page:vertical :disabled {
  background: #14506E;
}

QSlider::sub-page:horizontal {
  background: #1464A0;
  border: 1px solid #32414B;
  height: 4px;
  margin: 0px;
  border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
  background: #14506E;
}

QSlider::handle:horizontal {
  background: #787878;
  border: 1px solid #32414B;
  width: 8px;
  height: 8px;
  margin: -8px 0px;
  border-radius: 4px;
}

QSlider::handle:horizontal:hover {
  background: #148CD2;
  border: 1px solid #148CD2;
}

QSlider::handle:vertical {
  background: #787878;
  border: 1px solid #32414B;
  width: 8px;
  height: 8px;
  margin: 0 -8px;
  border-radius: 4px;
}

QSlider::handle:vertical:hover {
  background: #148CD2;
  border: 1px solid #148CD2;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Выберите метод:</string>
    </property>
    <widget class="QRadioButton" name="newton_cotesButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>250</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton::indicator::unchecked
{
   image: url(/Users/Horatio/source/Integral_3_ways-CourseWork1/img/circle.png);
}
QRadioButton::indicator::checked 
{
    image: url(/Users/Horatio/source/Integral_3_ways-CourseWork1/img/checked.png);
}</string>
     </property>
     <property name="text">
      <string>  Ньютона-Котеса</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="averageButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>33</y>
       <width>250</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton::indicator::unchecked
{
   image: url(/Users/Horatio/source/Integral_3_ways-CourseWork1/img/circle.png);
}
QRadioButton::indicator::checked 
{
    image: url(/Users/Horatio/source/Integral_3_ways-CourseWork1/img/checked.png);
}</string>
     </property>
     <property name="text">
      <string>  Средних</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="simpsonButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>64</y>
       <width>250</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton::indicator::unchecked
{
   image: url(/Users/Horatio/source/Integral_3_ways-CourseWork1/img/circle.png);
}
QRadioButton::indicator::checked 
{
    image: url(/Users/Horatio/source/Integral_3_ways-CourseWork1/img/checked.png);
}</string>
     </property>
     <property name="text">
      <string>  Симпсона</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>532</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>14</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string>Выберите точность:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>46</y>
       <width>245</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider:focus {
  border: none;
}

QSlider::groove:horizontal {
  background: #9b9b9b;
  border: 1px solid #32414B;
  height: 4px;
  margin: 0px;
  border-radius: 4px;
}


QSlider::sub-page:horizontal:disabled {
  background: #14506E;
}

QSlider::handle:horizontal {
  background: #2962ff;
  border: 1px solid #32414B;
  width: 16px;
  height: 8px;
  margin: -8px 0px;
  border-radius: 9px;
}

QSlider::handle:horizontal:hover {
  background: #2251d2;
  
}

</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="step">
     <property name="geometry">
      <rect>
       <x>269</x>
       <y>46</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>14</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
	color: #9b9b9b;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>701</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QChartView" name="graphic">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>66</y>
       <width>601</width>
       <height>366</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>513</x>
       <y>465</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: #9b9b9b
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="answer">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>470</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: #9b9b9b
}</string>
     </property>
     <property name="text">
      <string>Ответ:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LOAD">
     <property name="geometry">
      <rect>
       <x>307</x>
       <y>462</y>
       <width>90</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	color: #ebebeb;
	background: #2962ff;
	border-radius: 5px;
}
QPushButton:hover:!pressed
{
	background: #2251d2;
	
}
QPushButton:pressed
{
	background: #173894;
	
}</string>
     </property>
     <property name="text">
      <string>LOAD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SAVE">
     <property name="geometry">
      <rect>
       <x>207</x>
       <y>462</y>
       <width>90</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	color: #ebebeb;
	background: #2962ff;
	border-radius: 5px;
}
QPushButton:hover:!pressed
{
	background: #2251d2;
	
}
QPushButton:pressed
{
	background: #173894;
	
}</string>
     </property>
     <property name="text">
      <string>SAVE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="right">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>462</y>
       <width>70</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	color: #ebebeb;
	background: #2962ff;
	border-radius: 5px;
}
QPushButton:hover:!pressed
{
	background: #2251d2;
	
}
QPushButton:pressed
{
	background: #173894;
	
}</string>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="left">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>462</y>
       <width>70</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	color: #ebebeb;
	background: #2962ff;
	border-radius: 5px;
}
QPushButton:hover:!pressed
{
	background: #2251d2;
	
}
QPushButton:pressed
{
	background: #173894;
	
}</string>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>520</y>
      <width>491</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_A">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>13</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>A:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="aLineEdit">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>14</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_B">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>13</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>B:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="bLineEdit">
      <property name="geometry">
       <rect>
        <x>54</x>
        <y>14</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_C">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>13</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>C:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cLineEdit">
      <property name="geometry">
       <rect>
        <x>76</x>
        <y>14</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_D">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>13</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>D:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dLineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>14</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="equalButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>554</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color: #ebebeb;
	background: #2962ff;
	border-radius: 5px;
}
QPushButton:hover:!pressed
{
	background: #2251d2;
	
}
QPushButton:pressed
{
	background: #173894;
	
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>220</y>
      <width>415</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="nextFunButton">
     <property name="geometry">
      <rect>
       <x>336</x>
       <y>40</y>
       <width>75</width>
       <height>108</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover:!pressed
{
	
	  border: none;
	background-color:rgb(227, 227, 227);
}
QPushButton:pressed
{
  border: none;
	background-color:rgb(41, 97, 252);
}</string>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="prevFunButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>75</width>
       <height>108</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover:!pressed
{
	
	  border: none;
	background-color:rgb(227, 227, 227);
}
QPushButton:pressed
{
  border: none;
	background-color:rgb(41, 97, 252);
}</string>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>30</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGraphicsView{
background-image : &quot;/Users/Horatio/source/Integral_3_ways-CourseWork/img/integral-GUI.png&quot;
}</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
